mw-webapp/src/logic/wayPage/reportsTable/DayReportsTable.tsx

```diff
@@ -65,6 +66,15 @@ export const DayReportsTable = (props: DayReportsTableProps) => {
     setDayReports(dayReportsList);
   };
 
+  /**
+   * Check date
+   */
+  const isDateExist = (report: DayReport) => {
+    return report
```

Comment:
How does it wor? It shouldnalways return true

Reply:
Deleted function and created const isPossibleToCreateDayReport